File: Service/GroupMembershipManagement/Hosts/WebApi/Services.Messages/Requests/GetOrgLeaderDetailsRequest.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

using Models;
using Services.Messages.Contracts.Requests;
using System;

namespace Services.Messages.Requests
{
    public class GetOrgLeaderDetailsRequest : RequestBase
    {
        public GetOrgLeaderDetailsRequest(string objectId)
        {
            ObjectId = objectId;
        }

        public string ObjectId { get; }
    }
}

File: UI/web-app/src/models/GetOrgLeaderDetailsRequest.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

export interface GetOrgLeaderDetailsRequest {
  objectId: string;
  key: number;
  text: string;
  partId: string;
}

export interface GetOrgLeaderDetailsUsingIdRequest {
  employeeId: number;
  partId: string;
};